doc/spec-files.rst

```diff
@@ -343,9 +343,32 @@ For example modify the spec file this way::
           exclude_binaries=...
           )
 
+One-File mode
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If you are hoping to build a one file mode, the ``COLLECTION`` should be an empty tuple with its previous values absorbed by the ``EXE`` class. Your ``.exe`` file will still be in the dist folder with the name you specify here as the ``name``
+
+	  exe = EXE(pyz,
+              a.scripts,
+              a.binaries,
+              a.zipfiles,
+              a.datas,
+              exclude_binaries=False,
+              name='install',
+              icon='icon.ICO',
+              debug=False,
+              strip=False,
+              upx=True,
+              console=True )
```

Comment:
IMHO it is more helpful to know what to change, so listing *all* options is distracting. Only show what needs to be changed. This is basically what you describe in words in the next paragraph.